src/stacks-hierarchy/Root_FareContractDetailsScreen.tsx

```diff
@@ -1,29 +1,43 @@
 import {FullScreenHeader, useTicketInfo} from '@atb/components/screen-header';
-import {DetailsContent} from '@atb/fare-contracts';
+import {CarnetDetailedView, DetailsContent} from '@atb/fare-contracts';
 import {useApplePassPresentationSuppression} from '@atb/suppress-pass-presentation';
 import {StyleSheet} from '@atb/theme';
-import {
-  FareContractTexts,
-  useTranslation,
-} from '@atb/translations';
+import {FareContractTexts, useTranslation} from '@atb/translations';
 import React from 'react';
 import {ScrollView, View} from 'react-native';
 import {RootStackScreenProps} from '../stacks-hierarchy/navigation-types';
 import {useTimeContextState} from '@atb/time';
 import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {useAnalytics} from '@atb/analytics';
 
 type Props = RootStackScreenProps<'Root_FareContractDetailsScreen'>;
 
+export type TicketType = 'normal' | 'carnet';
```

Comment:
I can suggest to align the ticket type to something more aligned, `normal` perhaps is a type that might not be on our wording.

This is what I have heard:

| 'PreActivatedSingleTicket'
    | 'PreActivatedPeriodTicket'
    | 'CarnetTicket'
    | 'NightTicket'
    | 'SingleBoatTicket'
    | 'PeriodBoatTicket'
    | 'UnknownTicket';
    
    There is something called `NormalTravelRight` 🤔

Reply:
Apparently we can use `fareContract` to find out if it is a carnet or not, so this type is no longer needed.